### PR TITLE
Some suggested changes to the trajectory tutorial

### PR DESCRIPTION
**Pull request by ml-evs**
Hi @JPBergsma, I've just had a read through the tutorial and have collected some suggestions below. As I used the GitHub web interface these are unfortunately bundled into a single commit, sorry about that! I'll try to add some comments on particular lines below.

### CODE REVIEW

File: docs/tutorial_trajectories.md [hunk 1]
```diff
@@ -144,39 +160,37 @@ You can use the same environment as before.
...
 Next, you can run `optimade-validator http://localhost:5000` to validate the setup of your database.
-At the moment, the validator may still give a "'StrictFieldInfo' object is not subscriptable" error. You can ignore this error for now.
-Errors under INTERNAL FAILURES indicate problems with the validator itself and not with the server setup. You can report those [here](https://github.com/JPBergsma/optimade-python-tools/issues).
-More details about validating your server can be found in [validation.md](https://github.com/JPBergsma/optimade-python-tools/blob/optimade_python_tools_trajectory_0.1/docs/concepts/validation.md).
 
-### Deployment
 
-If you are satisfied with how things are , you can run the server with:
-`uvicorn optimade.server.main:app --reload --port=5000 --host=0.0.0.0` for it to be reachable from any IPv4 adress and
-`uvicorn optimade.server.main:app --reload --port=5000 --host='::'` for IPv6
```
Comment (ml-evs) [inline docs/tutorial_trajectories.md:164]: I think discussing `--host` here is a bit confusing and "reachable from any IPv4" address is misleading I think as it is only an internal networking distinction. You also don't mention that the database must be visible from this server too. 

I think I would suggest adding a link to the container deployment docs at https://www.optimade.org/optimade-python-tools/latest/deployment/container/ as they should be generally useful.

File: docs/tutorial_trajectories.md [hunk 2]
```diff
@@ -1,73 +1,89 @@
...
 
 The tutorial explains how to set up an OPTIMADE server for trajectory data.
-The code linked to here matches the trajectory endpoint as described in [https://github.com/JPBergsma/OPTIMADE/blob/Trajectory_proposal_v0.1/optimade.rst](https://github.com/JPBergsma/OPTIMADE/blob/Trajectory_proposal_v0.1/optimade.rst) and discussed in [OPTIMADE PR#377](https://github.com/Materials-Consortia/OPTIMADE/pull/377):
+The code linked to here matches the trajectory endpoint as described in [the trajectories proposal](https://github.com/JPBergsma/OPTIMADE/blob/Trajectory_proposal_v0.1/optimade.rst) and discussed in [OPTIMADE PR#377](https://github.com/Materials-Consortia/OPTIMADE/pull/377).
 This corresponds to [version 1.1 of the OPTIMADE specification](https://github.com/Materials-Consortia/OPTIMADE/releases/tag/v1.1.0).
-The trajectory endpoint is however still under development and there will probably be significant changes before it is merged with the main OPTIMADE specification.  
+The trajectory endpoint is however still under development and there may still be significant changes before it is merged with the main OPTIMADE specification, based on feedback from the community.
```
Comment (JPBergsma) [inline docs/tutorial_trajectories.md]: "may still" is not strong enough. The chance that significant changes will be made is much larger than 50%. So I think I'll stick with "will probably" 

```suggestion
The trajectory endpoint is however still under development and there will probably be significant changes before it is merged with the main OPTIMADE specification, based on feedback from the community.
```

File: docs/tutorial_trajectories.md [hunk 3]
```diff
@@ -1,73 +1,89 @@
...
+Other backends can be used as well, but will require creating a custom filtertransformer for converting the query into a backend compatible format and a custom [`EntryCollection`](https://www.optimade.org/optimade-python-tools/latest/api_reference/server/entry_collections/entry_collections/) object for interacting with the database backend.
+More generic advice on setting up an OPTIMADE API can be found in the [online documentation](https://www.optimade.org/optimade-python-tools/latest/getting_started/setting_up_an_api/).
 
-### Acquiring the trajectory version of the optimade python tools
+In this tutorial, we will use a Ubuntu-based Linux distribution with MongoDB as the backend for our trajectory data.
+At the end of this document you can find a troubleshooting section.
 
-The first step is to install the optimade python tools.
-Here, we briefly describe the steps required to install the trajectory branch of the optimade python tools.
+### Acquiring the trajectory version of the optimade-python-tools
```
Comment (JPBergsma) [inline docs/tutorial_trajectories.md]:
```suggestion
### Acquiring the trajectory version of optimade-python-tools
```

If you drop the "the" in the line below you should also do it here.

File: docs/tutorial_trajectories.md [hunk 4]
```diff
@@ -1,73 +1,89 @@
 ## Setting up an OPTIMADE Trajectory database.
 
 The tutorial explains how to set up an OPTIMADE server for trajectory data.
-The code linked to here matches the trajectory endpoint as described in [https://github.com/JPBergsma/OPTIMADE/blob/Trajectory_proposal_v0.1/optimade.rst](https://github.com/JPBergsma/OPTIMADE/blob/Trajectory_proposal_v0.1/optimade.rst) and discussed in [OPTIMADE PR#377](https://github.com/Materials-Consortia/OPTIMADE/pull/377):
+The code linked to here matches the trajectory endpoint as described in [the trajectories proposal](https://github.com/JPBergsma/OPTIMADE/blob/Trajectory_proposal_v0.1/optimade.rst) and discussed in [OPTIMADE PR#377](https://github.com/Materials-Consortia/OPTIMADE/pull/377).
```
Comment (JPBergsma) [inline docs/tutorial_trajectories.md]:
```suggestion
The code linked to here matches the trajectory endpoint as described in [v0.1 of the trajectories proposal](https://github.com/JPBergsma/OPTIMADE/blob/Trajectory_proposal_v0.1/optimade.rst) and discussed in [OPTIMADE PR#377](https://github.com/Materials-Consortia/OPTIMADE/pull/377).
```

File: docs/tutorial_trajectories.md [hunk 5]
```diff
@@ -1,73 +1,89 @@
...
+```pip install -e .[server]```
 
 ### Installing MongoDB
 
-The installation instructions for MongoDB can be found at [https://www.mongodb.com/docs/manual/installation/](https://www.mongodb.com/docs/manual/installation/)
-The community edition is good enough for our purpose.
-To automatically run the mongod daemon when the machine is booted, you can run: `systemctl enable mongod.service`
+The installation instructions for MongoDB can be found on the [MongoDB website](https://www.mongodb.com/docs/manual/installation/)
+The free community edition is good enough for our purposes.
+To automatically run the `mongod` daemon when the machine is booted, you can run: `systemctl enable mongod.service`, or simply launch it in a given terminal with `mongod`.
```
Comment (JPBergsma) [inline docs/tutorial_trajectories.md]:
```suggestion
To automatically run the `mongod` daemon when the machine is booted, you can run: `systemctl enable mongod.service`, to run it just once you can use: `systemctl start mongod.service`.
```
`mongod` does not seem to work when I run it in my virtual machine. I get the error that the directory data/db is missing. I do not get the error when I use systemctl.

File: docs/tutorial_trajectories.md [hunk 6]
```diff
@@ -1,73 +1,89 @@
...
 ### Set up the config file
 
-The next step is setting up the configuration file ".optimade.json".
-The default location is in the user's home directory, i.e "~/.optimade.json".
-More information and alternative ways for setting up the configuration parameters can be found in the [configuration.md](https://github.com/JPBergsma/optimade-python-tools/blob/optimade_python_tools_trajectory_0.1/docs/configuration.md) file.
-An example configuration file "optimade_config.json" is contained in the package and can be used as starting point for creating your own configuration file.
-If you are setting up a new database backend, the important parameters to set are:
+The next step is setting up the server configuration file.
+The default location is in the user's home directory, i.e `"~/.optimade.json"`.
+More information and alternative ways for setting up the configuration parameters can be found in the [online configuration instructions](https://www.optimade.org/optimade-python-tools/latest/configuration/).
```
Comment (JPBergsma) [inline docs/tutorial_trajectories.md]:
```suggestion
More information and alternative ways for setting up the configuration parameters can be found in the [online configuration instructions](https://github.com/JPBergsma/optimade-python-tools/blob/optimade_python_tools_trajectory_0.1/docs/configuration.md).
```

File: docs/tutorial_trajectories.md [hunk 7]
```diff
@@ -144,39 +160,37 @@ You can use the same environment as before.
...
 
-### Deployment
 
-If you are satisfied with how things are , you can run the server with:
-`uvicorn optimade.server.main:app --reload --port=5000 --host=0.0.0.0` for it to be reachable from any IPv4 adress and
-`uvicorn optimade.server.main:app --reload --port=5000 --host='::'` for IPv6
-By adding the --reload flag, the server is automatically restarted when the code of the optimade python tools is changed.
+At the moment, the validator may still give a `"'StrictFieldInfo' object is not subscriptable"` error, which can be safely ignored for now.
+Any errors under INTERNAL FAILURES indicate problems with the validator itself and not with the server setup. You can report those [here](https://github.com/JPBergsma/optimade-python-tools/issues).
+More details about validating your server can be found in [the online documentation](https://www.optimade.org/optimade-python-tools/latest/concepts/validation/).
```
Comment (JPBergsma) [inline docs/tutorial_trajectories.md]:
```suggestion
More details about validating your server can be found in [the online documentation](https://github.com/JPBergsma/optimade-python-tools/blob/optimade_python_tools_trajectory_0.1/docs/concepts/validation.md).
```

File: docs/tutorial_trajectories.md [hunk 8]
```diff
@@ -1,73 +1,89 @@
...
+```conda create -n optimade-traj python=3.10```
+
+You could also use Python versions 3.8 or 3.9 if you need to integrate with other libraries that require them. 
+You can then activate and begin using the Conda environment with: `conda activate optimade-traj`
 
 
 ### Install the trajectory version of the optimade python tools
 
-Next, you can install the local version of this package by going into the optimade-python-tools folder and running `pip install -e .[server]`.
+Next, you can install the local version of this package by going into the optimade-python-tools folder created during the `git clone` and installing the package locally with:
```
Comment (JPBergsma) [inline docs/tutorial_trajectories.md]:
```suggestion
Next, you can install the local version of this package by going into the optimade-python-tools folder, created during the `git clone`, and installing the package locally with:
```

File: docs/tutorial_trajectories.md (hunk 1 above)
Comment (JPBergsma) [reply docs/tutorial_trajectories.md:164]: The trajectory branch uses python packages that are not in the standard optimade container. (h5py and NumPy) 
So I think I should create a separate container image that already includes these python packages, or at least give instructions on how to install them in the docker container, before I reference the instructions for the docker container.

Comment (JPBergsma) [reply docs/tutorial_trajectories.md:164]: I have added an extra section to explain how you can run the trajectory version of the optimade-python-tools in a docker container. I ran into some problems when I tried to use the local code in a docker container(see issue #1589), so instead I let them create their own container.

**Pull request merged**